Transbank/WebpayRest/Oneclick/Request/StartRequest.cs

```diff
@@ -0,0 +1,33 @@
+﻿using Newtonsoft.Json;
+using Transbank.Webpay.Common;
+using System.Net.Http;
+
+namespace Transbank.Webpay.Oneclick.Requests
+{
+    internal class StartRequest : BaseRequest
+    {
+        [JsonProperty("username")]
+        public string UserName { get; set; }
```

Comment:
The same as before: Properties' access level should be equal or stricter than the Class access level